site/snippets/layouts/menu-items.php

```diff
@@ -0,0 +1,9 @@
+<?php foreach ($items as $label => $target): ?>
+	<?php if ($target instanceof \Kirby\Cms\Page): ?>
+	<li><a href="<?= $target->menuUrl() ?>"><?= $label ?></a></li>
+	<?php elseif (is_string($label) === true): ?>
+	<li><a <?= e($externalClass ?? null, 'class="' . $externalClass . '" ') ?>href="<?= $target ?>"><?= $label ?></a></li>
```

Comment:
`Issue:` `$externalClass` is never passed where the snippet is used. I would actually suggest to drop that variable and always add the class `is-external` here.

```suggestion
	<li><a class="is-external" href="<?= $target ?>"><?= $label ?></a></li>
```

Reply:
It is (in the main menu). We cannot hardcode the class as the footer menu shouldn't have it.

The issue was caused by the use of `e()` that makes the string in the second argument already evaluate even if the variable is empty or doesn't exist (unlike the `if` we had before). Fixed it with another fallback. But TBH, by that point I would prefer to go back to the original version with the `if`.

Reply:
Why shouldn't the footer menu have it?

Reply:
It probably doesn't harm, but we don't use the class for anything there. In the main menu it controls the arrow icon, the footer doesn't have that.

Reply:
Yes, that's why I think it would be beneficial to just include it for the footer as well. It simplifies the logic in the `menu-item` snippet quite nicely and doesn't affect (the look of) the footer currently. But if we ever would want to differentiate those, this class would also make sense in the context of the footer.